webplugin/js/app/mck-sidebox-1.0.js

```diff
@@ -65,7 +65,8 @@ var KM_ATTACHMENT_V2_SUPPORTED_MIME_TYPES = ["application","text","image"];
             'defaultChatDisabledMessage': 'Chat Disabled!'
         },
         voiceInput:false,
-        voiceOutput:false
+        voiceOutput:false,
+        attachment:false
```

Comment:
@Prochnost55 We can remove this.
We need to keep this setting "true" by default which is already being done in below line.
var MCK_ATTACHMENT = (typeof appOptions.attachment === "boolean") ? appOptions.attachment : true;

In Dashboard also, if nothing is coming in chatWidget object then treat it as "true".

Reply:
Okay, sir, I have done the required changes.